symphony-bdk-core/src/main/java/com/symphony/bdk/core/activity/ActivityRegistry.java

```diff
@@ -28,9 +31,37 @@
   /** The bot session forwarded to command-based activities only */
   private final UserV2 botSession;
 
+  private final MessageService messageService;
+
+  private SlashCommand helpCommand;
+
   /** The Datafeed real-time events source, or Datafeed listener */
   private final Consumer<RealTimeEventListener> realTimeEventsSource;
 
+  public ActivityRegistry(UserV2 botSession, Consumer<RealTimeEventListener> realTimeEventsSource , MessageService messageService) {
+    this.botSession = botSession;
+    this.realTimeEventsSource = realTimeEventsSource;
+    this.messageService = messageService;
+
+    this.registerHelpCommand();
+  }
+
+  private void registerHelpCommand() {
+    this.helpCommand = SlashCommand.slash("/help", commandContext -> {
```

Comment:
What happens if another `/help` command is registered by bot developer?

Reply:
I didn't think about it. Maybe we should have a Map to store all the commands with the command name?

Reply:
Maybe a more general question: what happens if a developer override an existing slash command already defined in his project? 

I'd say: skip it or crash. But any developer should have the ability to override the `/help` command for sure

Reply:
We should let developers override already defined /slash commands IMHO (and even unregister existing ones too in case you don' want the help command)